testutil-server/src/test/java/io/spine/testing/server/blackbox/given/BbProjectAggregate.java

```diff
@@ -87,6 +99,33 @@ BbTaskAdded handle(BbAddTask command) throws BbTaskCreatedInCompletedProject {
                 .build();
     }
 
+    @Assign
+    BbUserAssigned handle(BbAddProjectAssignee command) {
+        return BbUserAssignedVBuilder
+                .newBuilder()
+                .setId(getId())
+                .setUserId(command.getUserId())
+                .build();
+    }
+
+    @React(external = true)
+    Optional<BbUserUnassigned> on(BbUserDeleted event) {
```

Comment:
`BbUserUnassigned` is about a user, while we speak of a `Project` aggregate. So probably this is `BbAssigneeRemoved`.